tutorials/prometheus-monitoring-grafana-dashboard/index.mdx

```diff
@@ -24,29 +24,28 @@ This tutorial explains how to use a Prometheus Monitoring server with a Grafana
 
 - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
 - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key/)
+- You have a [Scaleway Instance](/compute/instances/how-to/create-an-instance/) running on [Ubuntu Focal Fossa (20.04)](https://www.ubuntu.com/)
 
 </Message>
 
 ## Preparing your environment
 
-In this tutorial, we use an Instance running on [Ubuntu Focal Fossa (20.04))](https://www.ubuntu.com/).
-
-1. To run Prometheus safely on our server, we have to create a user for Prometheus and Node Exporter without the possibility to log in.
+1. We have to create a user for Prometheus and Node Exporter without the possibility to log in.
```

Comment:
Could this explanatory bit all go in the text before step one, and then step 1 be "Run the following command to create the user":